default.nix

```diff
@@ -21,30 +22,51 @@ let
     # the Haskell.nix package set, reduced to local packages.
     (selectProjectPackages cardanoBenchmarkingHaskellPackages);
 
-  self = {
-    inherit haskellPackages cardanoNode pkgs;
+  scripts = callPackage ./nix/scripts.nix { inherit customConfig; };
+
+  rewrite-static = _: p: if (pkgs.stdenv.hostPlatform.isDarwin) then
+    pkgs.runCommandCC p.name {
+      nativeBuildInputs = [ pkgs.haskellBuildUtils.package pkgs.buildPackages.binutils pkgs.buildPackages.nix ];
+    } ''
+      cp -R ${p} $out
+      chmod -R +w $out
+      rewrite-libs $out/bin $out/bin/*
+    '' else if (pkgs.stdenv.hostPlatform.isMusl) then
+    pkgs.runCommandCC p.name { } ''
+      cp -R ${p} $out
+      chmod -R +w $out
+      $STRIP $out/bin/*
+    '' else p;
```

Comment:
I have no idea what that is?
These are the magic stub files that we just copy over from the nix-guys ?
Why do we have to deal with Darwin and Musl ?

Reply:
I've cut off a _lot_ of darwin/musl stuff from the `cardano-node` blueprint, but that still slipped through.

Thank you Marc!